### PR TITLE
Added the change directory command in the documentation

### LINKED ISSUE #231: Add: Change directory command in the official documentation
#### Add the change directory command on in the official documentation so after cloning the repository the user can directly operate the folder from the terminal

### PR DESCRIPTION
**Pull request by hasnainmakada-99**
> First thing, PLEASE READ THIS: [ReactPlay Code Review Checklist](https://github.com/atapas/react-play/wiki/ReactPlay-Code-Review-Checklist)

# Description
Added the change directory command in the official documentation so the users can directly operate it from the terminal

# Fixes #231 

## Type of change

Please delete options that are not relevant.

- [x] This change requires a documentation update

# How Has This Been Tested?

Seen the readme on Vs Code

# Checklist:
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings

### CODE REVIEW

**Comment by atapas** [comment]
@hasnainmakada-99 Please create an issue for this one.

File: README.md [hunk 1]
```diff
@@ -72,6 +72,8 @@ First, you need to fork the `react-play` repo. You can do this by clicking the `
 
 Once forked, you can clone the repo by clicking the `Clone or Download` button on the top right corner of forked repo.
 
+After cloning the repo open the terminal and type ```cd 'foldername'``` and you will be redirected to your forked repo.
```
Comment (atapas) [inline README.md:75]: I would put it as follows,

> Please change the directory after cloning the repository using the `cd <folder-name>` command.

**Review by atapas — CHANGES_REQUESTED** [review]

**Comment by hasnainmakada-99** [comment]
Sure tapas sir

File: README.md (hunk 1 above)
Comment (hasnainmakada-99) [reply README.md]: I've made the changes @atapas sir

**Comment by hasnainmakada-99** [comment]
Means sir ?

**Review by atapas — CHANGES_REQUESTED** [review]
Please update the string as I suggested.

File: README.md (hunk 1 above)
Comment (atapas) [reply README.md]: I do not see your changes.

**Comment by hasnainmakada-99** [comment]
> Please update the string as I suggested.

I've already updated that @atapas sir

**Comment by hasnainmakada-99** [comment]
> > Please update the string as I suggested.
> 
> I've already updated that @atapas sir

So sorry @atapas sir I've forgot to put a dash between folder - name. So sorry

**Comment by atapas** [comment]
> > > Please update the string as I suggested.
> > 
> > 
> > I've already updated that @atapas sir
> 
> So sorry @atapas sir I've forgot to put a dash between folder - name. So sorry

@hasnainmakada-99 This is what I see in the diff

image

But I had suggested this

Please change the directory after cloning the repository using the cd `<folder-name>` command.

**Comment by atapas** [comment]
> Yes sir it was my mistake I'll do it as you suggested

I still do not see your changes as I suggested. Could you please go to file changed tab and confirm?

@hasnainmakada-99

**Comment by hasnainmakada-99** [comment]
> > Yes sir it was my mistake I'll do it as you suggested
> 
> I still do not see your changes as I suggested. Could you please go to file changed tab and confirm? Mostly you are not pushing your changes, just committing.
> 
> @hasnainmakada-99

image

see sir here are the changes

**Comment by atapas** [comment]
> > > Yes sir it was my mistake I'll do it as you suggested
> > 
> > 
> > I still do not see your changes as I suggested. Could you please go to file changed tab and confirm? Mostly you are not pushing your changes, just committing.
> > @hasnainmakada-99
> 
> image
> 
> see sir here are the changes

I see that but that is not the text I suggested in my review comment above.

I had suggested this text: 

Please change the directory after cloning the repository using the cd `<folder-name>` command.

**Comment by hasnainmakada-99** [comment]
@atapas sir now I've updated that string

File: README.md [hunk 2]
```diff
@@ -72,6 +72,8 @@ First, you need to fork the `react-play` repo. You can do this by clicking the `
 
 Once forked, you can clone the repo by clicking the `Clone or Download` button on the top right corner of forked repo.
 
+Please change the directory after cloning the repository using the ```cd <folder-name>``` command.
```
Comment (atapas) [inline README.md:75]: Don't use three backticks. one is enough.

**Review by atapas — CHANGES_REQUESTED** [review]

File: README.md (hunk 2 above)
Comment (hasnainmakada-99) [reply README.md:75]: Ok

**Pull request merged**